_home/home.md

```diff
@@ -6,7 +6,7 @@ title: "Home"
 A Browser securing your privacy. That's it.
 ================================  
       
-[Download](downloads/){: .button .download}     
+[Download](downloads/){: .button .big .download}     
 	 
-The iridiumbrowser is based on the Chromium code base. All modifications enhance the privacy of the user and make sure that the latest and best secure technologies are used.
+The IridiumBrowser is based on the Chromium code base. All modifications enhance the privacy of the user and make sure that the latest and best secure technologies are used.
```

Comment:
this is how it was written from the beginning and I have continued to use it as one word...